drawee/src/main/java/com/facebook/drawee/generic/GenericDraweeHierarchy.java

```diff
@@ -421,13 +423,6 @@ public void setPlaceholderImage(Drawable drawable, ScaleType scaleType) {
     getScaleTypeDrawableAtIndex(PLACEHOLDER_IMAGE_INDEX).setScaleType(scaleType);
   }
 
-  /**
-   * @return true if there is a placeholder image set.
-   */
-  public boolean hasPlaceholderImage() {
-    return getParentDrawableAtIndex(PLACEHOLDER_IMAGE_INDEX) != null;
-  }
-
```

Comment:
Please just bring this method back to avoid breaking the public API in case anyone is using it. Then we can merge this.